recipes/crashpad/all/test_package/conanfile.py

```diff
@@ -0,0 +1,21 @@
+from conans import ConanFile, CMake, tools
+import os
+
+
+class TestPackageConan(ConanFile):
+    settings = "os", "compiler", "build_type", "arch"
+    generators = "cmake"
+
+    def build(self):
+        cmake = CMake(self)
+        cmake.configure()
+        cmake.build()
+
+    def test(self):
+        if not tools.cross_building(self.settings):
+            test_env_dir = "test_env"
+            tools.mkdir(test_env_dir)
+            bin_path = os.path.join("bin", "test_package")
+            handler_exe = "crashpad_handler.exe" if self.settings.os == "Windows" else "crashpad_handler"
```

Comment:
Afaik it should work without the `.exe`

Reply:
This is an argument to the test_package and needs to be an existing path.
I tested it without the `.exe` and it didn't work on Windows.